tools/bwa-mem_v1.cwl

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env cwl-runner
+
```

Comment:
Hello @phelelani, can you update the existing `bwa-mem.cwl` instead of creating a new file? Thanks!